Dockerfile

```diff
@@ -1,10 +1,26 @@
-FROM node:12.10.0-stretch
+FROM node:12.20.1-stretch
 
+# install browsers for (unit)-testing during build-phase
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends 'chromium=73.0.3683.75-1~deb9u1' 'firefox-esr=78.6.1esr-1~deb9u1' && \
```

Comment:
These versions are **way** too old for us to run our tests on :smile: 

We only (officially) support:

Chromium 88 & 89
Firefox 83 & 84
Safari 13 & 14

Reply:
Yes there old but these were the default versions if got, will investigate furthur

Reply:
The container is ubuntu/debian based so it always lacks behind a fair bit.
If this isn't resolved in our first round of upgrades we can skip it till someone runs into the problems. Most tests don't require a browser anyway.